editor/src/reveal/editor/reveal.clj

```diff
@@ -2,12 +2,14 @@
   (:require [clojure.main :as m]
             [clojure.string :as str]
             [dynamo.graph :as g]
+            editor.code.data
```

Comment:
Wrap in `[]`? Although, do we even need to require it?

Reply:
Wrapping in `[]` is only needed if you want to do something with that ns, e.g. alias it. We only need to require it so imports can work — if we don't require the ns, `editor.code.data.*` classes won't exist and imports will throw exceptions.

Reply:
Btw, I know some Clojure linters like clj-kondo warn on unused `[]`-wrapped requires, but don't warn on.. uhm.. naked requires, because naked require signals that the ns is required for side effects, which is the case here.

Reply:
Ah, I see.